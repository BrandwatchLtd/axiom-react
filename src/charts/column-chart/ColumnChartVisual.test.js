import React from 'react';
import renderer from 'react-test-renderer';
import { ColumnChartVisual } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <ColumnChartVisual { ...props }>
      Lorem
    </ColumnChartVisual>
  );
}

describe('ColumnChartVisual', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
