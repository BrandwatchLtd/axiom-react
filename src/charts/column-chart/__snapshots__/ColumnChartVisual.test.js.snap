// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColumnChartVisual renders with defaultProps 1`] = `
<div
  className="ax-column-chart__visual"
>
  Lorem
</div>
`;
